qb/test/insert.test.ts

```diff
@@ -186,7 +186,28 @@ test("insert type enforcement", async () => {
 
 test("optional sequence fields", async () => {
   const query = e.insert(e.Bag, {
-    stringsMulti: "asdf",
+    stringsMulti: ["asdf"],
```

Comment:
And we should probably allow setting non array values for multi properties anyway. The `multi` just makes the upper cardinality `Many` instead of `One`, so singletons are still ok.